### DISCUSSION TITLE
Allow for ranges for servings, ingredients etc.

### DISCUSSION THREAD

**Discussion by KBD20** [TandoorRecipes/recipes#2725] [Ideas]
Some recipes have things like "serves 4-6" or "6-8 chillies (depending on heat), I have accounted for with notes but it doesn't work for scaling - a recipe that is "4-6" with 8-12 chillis scaled to 4 will show 8-6 servings and 16-12 chillis currently.

What I thought of is for situations like this is have a "range" option, a user might put 4 in one field and 6 in a "range" option, internally this would be a multiplier (4 - up to 15x/150%) and rounding down to the nearest whole for servings (maybe with a + or something, and nearest either way for ingredients (or nearest fraction).

**Comment by smilerz** [comment]
A reasonable request. However adding ranges has been considered and rejected previously to keep complexity (which is already too high) down.
